### ISSUE TITLE
Multiple Submissions in F2F Task

### ISSUE THREAD

**Issue by RiteshTC** [topcoder-platform/legacy-challenge-processor#43]
Legacy Challenge processor at times seems to create 2 submissions for a Task , with same timestamp.

<img width="1054" alt="Screen Shot 2021-03-16 at 5 23 42 PM" src="https://user-images.githubusercontent.com/5771673/111304948-61cfb100-867c-11eb-8599-10b7888e4145.png">

Challenge ID: 30173502

**Comment by ThomasKranitsas** [comment]
@RiteshTC are you sure this is the right repo? We don't create submissions in the legacy-challenge-processor

**Comment by rootelement** [comment]
Yes, I believe you're looking for @sachin-maheshwari and the [legacy-submission-processor](https://github.com/topcoder-platform/legacy-submission-processor/issues)

**Comment by sachin-maheshwari** [comment]
@RiteshTC should be fixed in today's release.

**Comment by RiteshTC** [comment]
I am not sure, if this is the correct repo. (related to challenge and its processor)
I don't think its connected to submission api.

Closing a task doesn't involve create a submission from the user end.  No record is created for submission in Dynamo. 
I think the submission record is directly written in Informix, without routing through the submission api.

**Comment by RiteshTC** [comment]
https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30174386

Another example on a recent task. It generates duplicate payments as well for the submitter.

<img width="1086" alt="Screen Shot 2021-03-19 at 5 42 04 PM" src="https://user-images.githubusercontent.com/5771673/111778302-79a47080-88da-11eb-9c73-f1dadd92b311.png">
